app/src/main/java/com/nexenio/bleindoorpositioningdemo/ui/beaconview/chart/BeaconLineChart.java

```diff
@@ -317,72 +329,83 @@ protected void drawBeacons(Canvas canvas) {
 
     @Override
     protected void drawBeacon(Canvas canvas, Beacon beacon) {
+        List<RssiFilter> filterList = new ArrayList<>();
+        filterList.add(meanFilter);
+        filterList.add(armaFilter);
+        filterList.add(kalmanFilter);
+
         currentBeaconIndex = beacons.indexOf(beacon);
 
-        linePaint.setShader(createLineShader(getBeaconColor(beacon, coloringMode, currentBeaconIndex)));
-        linePaint.setAlpha(128);
+        for (RssiFilter filter : filterList) {
```

Comment:
By default, we don't want to visualize different filters for one beacon. Either create a new view extending this one or provide some way of configuration.